specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/preview/2024-01-01-preview/examples/Endpoint/getModels.json

```diff
@@ -0,0 +1,135 @@
+                "deprecationDate": "2024-03-01T00:00:00+00:00",
+                "capacity": {
+                  "minimum": null,
+                  "maximum": 10000,
+                  "step": null,
+                  "default": 120
+                },
+                "rateLimits": [
+                  {
+                    "count": 1.0,
+                    "renewalPeriod": 10.0,
+                    "rules": []
+                  },
+                  {
+                    "count": 1000.0,
+                    "renewalPeriod": 60.0,
+                    "rules": []
+                  }
+                ]
+              }
+            ],
+            "maxCapacity": 3,
+            "capabilities": {
+              "fineTune": "true",
+              "inference": "false",
+              "scaleType": "Manual",
+              "completion": "true",
+              "search": "true",
+              "FineTuneTokensMaxValue": "2000000000"
+            },
+            "finetuneCapabilities": {
+              "scaleType": "Manual,Standard",
+              "completion": "true",
+              "search": "true",
+              "FineTuneTokensMaxValue": "2000000000"
+            },
+            "deprecation": {
+              "fineTune": "2024-07-05T00:00:00+00:00",
+              "inference": "2024-07-05T00:00:00+00:00"
+            },
+            "systemData": {
+              "createdBy": "Microsoft",
+              "createdByType": null,
+              "lastModifiedBy": "Microsoft",
+              "lastModifiedByType": null
+            },
+            "format": "OpenAI",
+            "name": "ada",
+            "version": "1",
+            "source": null,
+            "callRateLimit": null
+          },
+          {
+            "baseModel": null,
+            "isDefaultVersion": false,
+            "skus": [
+              {
+                "name": "Standard",
+                "usageName": "OpenAI.Standard.babbage",
+                "deprecationDate": "2024-03-01T00:00:00+00:00",
+                "capacity": {
+                  "minimum": null,
+                  "maximum": 10000,
+                  "step": null,
+                  "default": 120
+                },
+                "rateLimits": [
+                  {
+                    "count": 1.0,
+                    "renewalPeriod": 10.0,
+                    "rules": []
+                  },
+                  {
+                    "count": 1000.0,
+                    "renewalPeriod": 60.0,
+                    "rules": []
+                  }
+                ]
+              }
+            ],
+            "maxCapacity": 3,
+            "capabilities": {
+              "fineTune": "true",
+              "inference": "false",
+              "scaleType": "Manual",
+              "completion": "true",
+              "search": "true",
+              "FineTuneTokensMaxValue": "2000000000"
+            },
+            "finetuneCapabilities": {
```

Comment:
>finetune [](http://example.com/codeflow?start=13&length=8)

you're not consistently using camelCase for fineTune

Reply:
will update

Reply:
Oh I guess I can't fix that this is some info we got from cognitive service where you can see from their swagger the name is like this, since we want to merge the two service and migrate the API call, I think we have to keep them align right now for smooth transition. I will create work item to rename this in a future version.

Reply:
https://github.com/Azure/azure-rest-api-specs/blob/922913c1cb2f30be7660333fff75dc808c3634fb/specification/cognitiveservices/resource-manager/Microsoft.CognitiveServices/preview/2023-10-01-preview/cognitiveservices.json#L5210C13-L5210C13